### PR TITLE
Moved project reference file request

### PR DESCRIPTION
**Pull request by deepsidhu85**
## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Moved getting of reference files for projects into the reference fil ajax controller.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~
~* [ ] Tests added (or description of how to test) for any new features.~
~* [ ] User documentation updated for UI or technical changes.~

### CODE REVIEW

File: src/main/webapp/resources/js/apis/projects/reference-files.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { setBaseUrl } from "../../utilities/url-utilities";
  */
 export async function getProjectReferenceFiles(projectId) {
   return await axios
-    .get(setBaseUrl(`/projects/${projectId}/settings/ajax/reference/all`))
+    .get(setBaseUrl(`ajax/reference-files/${projectId}/all`))
```
Comment (joshsadam) [inline src/main/webapp/resources/js/apis/projects/reference-files.js:11]: Same as your controller method, do we really need the `/all` at the end?  What might be a better pattern here is `/ajax/reference-files?projectId=4` or something similar.  That way we know exactly when we write that url that is for a project.  Food for thought.

**Review by joshsadam — CHANGES_REQUESTED** [review]
This looks great!  One question about the format of your URL.

**Review by joshsadam — APPROVED** [review]
Alright

File: src/main/webapp/resources/js/apis/projects/reference-files.js (hunk 1 above)
Comment (deepsidhu85) [reply src/main/webapp/resources/js/apis/projects/reference-files.js]: Nope that's a good idea. Updated in b0250f63048872cf049ac1a2c4d98926b2ed6072

**Pull request merged**